app/javascript/lib/validation/validate-file-content.js

```diff
@@ -308,6 +425,7 @@ export async function parseClusterFile(chunker, mimeType) {
   await chunker.iterateLines((line, lineNum, isLastLine) => {
     const parsedLine = parseLine(line, delimiter)
     issues = issues.concat(validateUniqueCellNamesWithinFile(parsedLine, isLastLine, dataObj))
+    issues = issues.concat(validateGroupColumnCounts(parsedHeaders, parsedLine, isLastLine, dataObj))
```

Comment:
For readability, if/when those suggested changes in the upstream PR are done and merged here:

```suggestion
    issues = issues.concat(validateGroupColumnCounts(headers, line, isLastLine, dataObj))
```